src/Controls/src/Core/Registrar.cs

```diff
@@ -271,6 +271,8 @@ static Registrar()
 			Registered = new Registrar<IRegisterable>();
 		}
 
+		public static IFontRegistrar FontRegistrar { get; } = new FontRegistrar();
```

Comment:
This again is not using the service provider (like the `FontManager` on `Forms`) because we need this ready _before_ the registrar runs. This is used by the `RegisterAll` method, which is the initial entry point to register the things. Not sure what the correct way to use the service provider in a static context.